$MirOS: ports/mailnews/pine/patches/patch-build,v 1.1 2005/12/15 14:24:20 tg Exp $

The first chunk must contain @ETC@ three times.

--- build.orig	Wed May  4 21:22:00 2005
+++ build	Thu Dec 15 13:45:24 2005
@@ -464,13 +464,13 @@ case $maketarg in
 	then
 	    cd "$PHOME"
 	    rm .pine.sedargs
-	    echo -n " -e s/\/usr\/local\/lib\/pine.conf/\/etc\/pine.conf/" > .pine.sedargs
-	    echo -n " -e s/\/usr\/local\/lib\/pine.info/\/etc\/pine.info/" >> .pine.sedargs
+	    echo -n " -e s/\/usr\/local\/lib\/pine.conf/@ETC@\/pine.conf/" > .pine.sedargs
+	    echo -n " -e s/\/usr\/local\/lib\/pine.info/@ETC@\/pine.info/" >> .pine.sedargs
 	    echo -n " -e s/\/usr\/lib\/sendmail/\/usr\/sbin\/sendmail/" >> .pine.sedargs
 	    echo -n " -e s/\/usr\/spool\/news/\/var\/spool\/news/" >> .pine.sedargs
 	    echo -n " -e s/\/usr\/spool\/mail/\/var\/spool\/mail/" >> .pine.sedargs
 	    echo -n " -e s/\/bin\/passwd/\/usr\/bin\/passwd/" >> .pine.sedargs
-	    PINEEXTRAC="'LOCALPINECFLAGS=-DSYSTEM_PINERC=\\\"/etc/pine.conf\\\" -DSYSTEM_PINERC_FIXED=\\\"/etc/pine.conf.fixed\\\" -DSYSTEM_PINE_INFO_PATH=\\\"/etc/pine.info\\\"'"
+	    PINEEXTRAC="'LOCALPINECFLAGS=-DSYSTEM_PINERC=\\\"@ETC@/pine.conf\\\" -DSYSTEM_PINERC_FIXED=\\\"@ETC@/pine.conf.fixed\\\" -DSYSTEM_PINE_INFO_PATH=\\\"@ETC@/pine.info\\\"'"
 	fi
         cd "$PHOME/pine"
         eval echo make "$makeargs" "$L1" "$L2" "$PINEEXTRAC" -f makefile.$pinetarg
